docs/user/dql/functions.rst

```diff
@@ -893,6 +982,100 @@ Example::
     +----------------------+------------------------------------------+
 
 
+DATETIME
+--------
+
+Description
+>>>>>>>>>>>
+
+Usage: datetime(datetime)/ datetime(date, to_timezone) Converts the datetime to a new timezone
+
+Argument type: DATETIME/STRING
+
+Return type map:
+
+DATETIME, STRING -> DATETIME
+
+DATETIME -> DATETIME
+
+Example::
+
+    os> SELECT DATETIME('2008-12-25 05:30:00+00:00', 'America/Los_Angeles')
+    fetched rows / total rows = 1/1
+    +----------------------------------------------------------------+
+    | DATETIME('2008-12-25 05:30:00+00:00', 'America/Los_Angeles')   |
```

Comment:
Just wondering does `convert_tz` support timezone name as well? Didn't see example above.

Reply:
Fixed in 1fbed3f609b91568ec6251064e8bbdeb9eed83b5